assets/style.css

```diff
@@ -0,0 +1,781 @@
+/* GLOBAL */
+.flex {
+  display: flex;
+}
+
+a {
+  text-decoration: none;
+}
+
+.grid {
+  display: grid;
+}
+
+ul {
+  list-style: none;
+}
+
+.relative {
+  position: relative;
+}
+
+.absolute {
+  position: absolute;
+}
+
+.orange {
+  color: #f42;
+}
+
+.white {
+  color: #fff;
+}
+
+/* NAVIGATION */
+
+.nav-bar,
+.latest-deals-heading,
+.footer-links {
+  justify-content: space-between;
+}
+
+.logo {
+  width: 100px;
+}
+
+.fa-search {
+  transform: scaleX(-1);
+  color: #555;
+}
+
+.fa-bars {
+  margin-left: 10px;
+  color: #f42;
+}
+
+.lens-menu {
+  display: block;
+}
+
+.width-1024,
+.width-1023,
+.showcase-logo,
+.mobile-showcase-content .release-date {
+  display: none;
+}
+
+/* SHOWCASE */
+.showcase-section {
+  margin-top: 10px;
+  grid-template-rows: 1fr;
+}
+
+.main-showcase,
+.aside-showcase {
+  background-image: url(./images/main-showcase-image.jpeg);
+  height: 163px;
+  color: #fff;
+  margin-bottom: 15px;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `color:#fff` properties in entire CSS file
- [ ] You already has in line #30 declared use it 

**Example:**

**CSS file**


**Html file**

```HTML
<div class="white">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_